output.go

```diff
@@ -51,7 +51,7 @@ type Output struct {
 }
 
 func New(params output.Params) (output.Output, error) {
-	extensionConfig, err := GetConsolidatedConfig(params.JSONConfig)
+	extensionConfig, err := GetConsolidatedConfig(params.JSONConfig, params.Environment, params.ConfigArgument)
```

Comment:
```suggestion
	extensionConfig, err := GetConsolidatedConfig(params.JSONConfig, params.Environment)
```

I've suggested removing the URL parameter in 
https://github.com/leonyork/xk6-output-timestream/pull/261/files#r1468902395 so don't forget to remove here where we call it!